docs/designs/rbac-in-oss/research.md

```
@@ -0,0 +1,190 @@
+...
+if is_ee():
+    has_permission = await check_action_access(
+        user_uid=..., project_id=..., permission=Permission.VIEW_API_KEYS,
+    )
+    if not has_permission:
+        raise HTTPException(403, ...)
+```
+
+The 28 files:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)